crates/client/data-availability/src/eigenda/blob.rs

```
@@ -0,0 +1,182 @@
+// this is a copy of types in the eigenda disperser API: https://github.com/Layr-Labs/eigenda/blob/master/api/proto/disperser/disperser.proto
+
+use serde::{Serialize, Deserialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct DisperseBlobResponse {
+    status: BlobStatus,
+    #[serde(rename = "requestId")]
+    request_id: String
+}
+impl DisperseBlobResponse {
+    pub fn status(&self) -> &BlobStatus {
+        &self.status
+    }
+    pub fn request_id(&self) -> String {
+        self.request_id.clone()
+    }
+}
+impl Default for DisperseBlobResponse {
+    fn default() -> Self {
+        DisperseBlobResponse { status: Default::default(), request_id: Default::default() }
+    }
+}
+impl From<String> for DisperseBlobResponse {
+    fn from(value: String) -> Self {
+        if let Some(start_index) = value.find('{') {
+            let json_str = &value[start_index..];
+            println!("{}", &json_str);
+
+            let blob_response: Result<DisperseBlobResponse, serde_json::Error> = serde_json::from_str(json_str);
+
+            match blob_response {
+                Ok(response) => {
+                    return response
+                }
+                Err(err) => {
+                    println!("{}", &err);
+                    return DisperseBlobResponse::default() 
+                }
+            }
+        } else {
+            return DisperseBlobResponse::default()
```

instead of returning default should we do `TryFrom` and throw an error?